.github/workflows/development.yml

```diff
@@ -66,23 +65,23 @@ jobs:
         run: git diff --exit-code
 
       - name: Run tests
-        if: matrix.node != '12' || matrix.os != 'ubuntu-latest'
+        if: matrix.os != 'ubuntu-latest'
```

Comment:
This needs to be pointed at a different version, as it's used to make sure certain tasks are only run once

Reply:
Ah, ok, thanks. I’ll change it to 14 then.